go.mod

```diff
@@ -12,6 +12,7 @@ require (
 
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/canonical/sqlair v0.0.0-20241004123011-77313b5382fd
```

Comment:
It's strange that they don't really version sqlair.

Reply:
:man_shrugging: